go-controller/pkg/types/const.go

```diff
@@ -115,6 +115,10 @@ const (
 	OvnK8sTopoAnno         = OvnK8sPrefix + "/" + "topology-version"
 	OvnK8sSmallMTUTaintKey = OvnK8sPrefix + "/" + "mtu-too-small"
 
+	// name of the configmap used to synchronize status (e.g. watch for topology changes)
+	OvnK8sStatusCMName         = "status"
```

Comment:
we could reuse the ovn-config map as well and add a new field to it?

Reply:
We want to use our own config map, not ovn-config, so that we avoid duelling updates. When multiple controllers fight over the same object, it gets ugly.

I'll change the name to "control-plane-status" though.